### ISSUE TITLE
[Storybook] expose data used in each example

### ISSUE THREAD

**Issue by drewdaemon** [elastic/elastic-charts#2348]
We allow users to see the code used in each storybook example. However, they cannot see what data is actually generated and passed to the example. This would be helpful for understanding how the example works and how they should set up their own code.

**Comment by nickofthyme** [comment]
Yup great point! I think adding a blob somewhere in the story of all the data being used would be great. Also having a way to copy the code sufficient to render the chart outside of storybook.
